index.js

```diff
@@ -11,6 +11,7 @@ const colors = {
   blue: '#7FDBFF',
   lightGray: '#DDDDDD',
   gray: '#AAAAAA',
+  orange: '#FF851B',
```

Comment:
Think this looks good enough - if we really want we could add like a dark background to the names?

<img width="1274" alt="screen shot 2016-07-20 at 15 08 49" src="https://cloud.githubusercontent.com/assets/2467194/16987335/f0e31d24-4e8b-11e6-836b-c0f1dabe318f.png">